src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -68,14 +70,14 @@ def insar_tops_burst(
     )
     results = asf_search.search(opts=opts)
 
-    ref_not_found = reference_scene not in results.__str__()
-    sec_not_found = secondary_scene not in results.__str__()
+    ref_not_found = reference_scene not in str(results)
+    sec_not_found = secondary_scene not in str(results)
     if ref_not_found and sec_not_found:
-        raise RuntimeError(f'ASF Search failed to find both {reference_scene} and {secondary_scene}.')
+        raise ASFSearchError(f'ASF Search failed to find both {reference_scene} and {secondary_scene}.')
     elif ref_not_found:
-        raise RuntimeError(f'ASF Search failed to find {reference_scene}.')
+        raise ASFSearchError(f'ASF Search failed to find {reference_scene}.')
     elif sec_not_found:
-        raise RuntimeError(f'ASF Search failed to find {secondary_scene}.')
+        raise ASFSearchError(f'ASF Search failed to find {secondary_scene}.')
```

Comment:
I don't think we should explicitly raise `ASFSearchError`, as I believe that's an error that only the `asf_search` module itself should raise. If we can't think of a more appropriate exception, [`ValueError`](https://docs.python.org/3/library/exceptions.html#ValueError) is probably a safe bet.